### PR TITLE
add docker-compose project

### LINKED ISSUE #3: Setup Docker Compose for Django and Postgres

### PR DESCRIPTION
**Pull request by drewatk**
Add docker setup to README.md

Resolve #3

### CODE REVIEW

File: members/settings.py [hunk 1]
```diff
@@ -75,12 +75,11 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': 'directory',
-        'USER': 'directory_user',
-        'PASSWORD': 'TAF_4each',
-        'HOST': 'localhost',
-        'PORT': '',
+        'ENGINE': 'django.db.backends.postgresql',
```
Comment (thesephist) [inline members/settings.py:78]: @drewatk Were these changes made by docker? I think this is still correct, just curious.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,10 @@
 # taf-members
 Members site for Tippicanoe Arts Federation.
+
+
+## Instructions to run
+1. Install docker and compose
+2. Run `docker-compose up`. The server should be running on port 8000
+3. To run a database migration: `docker-compose run web python manage.py migrate`
+
```
Comment (thesephist) [inline README.md:9]: @drewatk To start the server, we still need to run `manage.py runserver`, right? That might be good to add.

I think ultimately we want the whole thing behind Nginx somewhere. But that might just be necessary when deployed, not necessary in docker.

**Review by thesephist — APPROVED** [review]
Looks Good To Me. See two minor comments / questions...

File: members/settings.py (hunk 1 above)
Comment (drewatk) [reply members/settings.py:78]: In the readme I linked the docker-compose guide I used

File: README.md (hunk 2 above)
Comment (drewatk) [reply README.md:9]: check out docker-compose.yml
That command is run by the web service being launched (`docker-compose up`)

**Pull request merged**